config/iao.yml

```diff
@@ -10,30 +10,35 @@ base_url: /obo/iao
 base_redirect: https://github.com/information-artifact-ontology/IAO 
 
 products:
-###  2017-03-24 is now current. There is no current OBO version. That should be fixed.
-- iao.owl: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/releases/2017-03-24/iao-merged.owl
-- iao.obo: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/releases/2009-11-06/merged/iao.obo
+- iao.owl: https://raw.githubusercontent.com/information-artifact-ontology/IAO/v2019-06-07/iao.owl
+- iao.obo: http://ontologies.berkeleybop.org/iao.obo
 
 term_browser: ontobee
 
 example_terms:
 - IAO_0000030
 
+
 entries:
 
-- exact: /iao-stated.owl
-  replacement: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/releases/2017-03-24/iao.owl
+#- exact: /iao-stated.owl
+#  replacement: https://raw.githubusercontent.com/information-artifact-ontology/IAO/master/releases/2017-03-24/iao.owl
```

Comment:
We should not drop PURLs if we can help it. I don't know why this PURL as made, but it should probably point to `iao.owl` in the latest release.